examples/testing/README.md

```diff
@@ -0,0 +1,105 @@
+Instructions For Running The Testing Examples
+=============================================
+
+The examples included here are used as automated tests, but they can also be explored manually.
+The fisrt example is very simple and subsequent examples are progressively more complex,
+so you can use these examples as an introduction into configuring StoneWork.
+
+It is recommended to read the [Getting started examlpe][getting-started] before proceeding to these examples.
+
+This document describes common prerequisities and instructions for running the examples.
+Specific information about individual examples can be found in their respective README files.
+
+Prerequisities
+--------------
+
+1. 
+   Make sure that `stonework` production image and testing-only image `tester` are already built.
+   In the top-level directory of the repository, trigger the makefile target `images` with:
+   ```
+   $ make images
+   ```
+2. 
+   **Optional**: StoneWork can be managed through `agentctl`, a CLI provided by the Ligato framework.
+   You can install it with
+   ```
+   $ go get go.ligato.io/vpp-agent/v3/cmd/agentctl
```

Comment:
The installation should, instead of `go get`, use `go install`, which is **the recommended way to install packages** since Go 1.16 (we [currently require Go 1.17](https://github.com/PANTHEONtech/StoneWork/blob/main/go.mod#L3)).

From [Go 1.16 Release Notes](https://tip.golang.org/doc/go1.16#go-command):

> go install, with or without a version suffix (as described above), is now the recommended way to build and install packages in module mode

Reply:
This will be done in #10